pkgs/top-level/all-packages.nix

```diff
@@ -15962,6 +15962,8 @@ with pkgs;
     inherit (darwin.apple_sdk.frameworks) Security;
   };
 
+  snarkvm = callPackage ../applications/blockchains/snarkvm { };
```

Comment:
Don't add new packages to `all-packages.nix` - use the `pkgs/by-name` paradigm instead (see `pkgs/by-name/README.md`)